create_book.rb

```diff
@@ -1,14 +1,20 @@
 require_relative 'book'
+require 'json'
 
 module CreateBook
   class << self
     attr_accessor :books
   end
 
-  @books = []
+  # reading books from json file methods
+  def self.load_books
+    base = "#{Dir.pwd}/saved_data"
+    books_reader = File.read("#{base}/books.json")
+    return if books_reader == ''
```

Comment:
- [ ] I noticed that the app preserves and loads the data just fine if the files exist. However, the moment I delete one of the files, I am faced with an error while either trying to create a book or exit the app.
Please take a look at the error below and try to fix it 👇 

![image](https://user-images.githubusercontent.com/46839548/220957661-1b4d1517-a98f-4a6d-9035-f34a1b4c3241.png)

Please handle the case where one or more files are missing. The app should work normally 👍